### ISSUE TITLE
Suggestion: Add Chance of Random Research Removal

### ISSUE THREAD

**Issue by ZEROOblivious** [Slimefun-Addon-Community/HardcoreSlimefun#4]
Greetings,

Just reading through the description, and could it be possible to add a percentage chance of losing random research on death? 

Also is it possible to add in specific permission nodes so we could increase/decrease the values per Server Premium Ranks? 
per say; you level up in-game with a prestige system, you have a lower chance of random loss/failed research percentages?

**Comment by Sefiraat** [comment]
You want to allow people to pay money for premium ranks to be ‘punished’ less by this addon, which you added to the server?

**Comment by ZEROOblivious** [comment]
No, No. Its not a money ranking system. You earn your prestige levels by completing tasks like killing X-Amount of mobs and such.

My ranks are not designed to be that way. 

but i want it to be those who've grinded their way to the highest level, have less of a chance of research loss.

**Comment by Sefiraat** [comment]
Hmm, Server Premium Ranks tend to mean something very specific, though benefit of the doubt I guess

**Comment by reamianv** [comment]
plz!
